R/cache.R

```diff
@@ -8,8 +8,9 @@ Cache <- R6::R6Class(
   "Cache",
   public = list(
     type = NULL,
-    initialize = function(type) {
+    initialize = function(type = NULL) {
       self$type <- type
+      private$cachedObjects <- new.env(hash = TRUE, parent = emptyenv())

```

Comment:
@PavelBal This was a massive bug here. The cache was only initialized when the class def was created (e.g. global) but not on every instance. Just spent two hours on that :) Glad I found it

Reply:
Wow :scream: